coingecko.py

```diff
@@ -3,14 +3,14 @@
 from typing import Optional
 
 
-def get_token_price_in_usd(address: str) -> Optional[float]:
+def get_token_price_in_usd(token_id: str) -> Optional[float]:
     """
     Returns the Coingecko price in usd of the given token.
     """
+    #token_id = "gnosis"
     coingecko_url = (
-        "https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses="
-        + address
-        + "&vs_currencies=usd"
+        "https://api.coingecko.com/api/v3/coins/"
+        + token_id
```

Comment:
Can you add a description for why you changed the api here? Does it return the same values as before when querying for `"ethereum"` or `"cow-protocol"`?